[libimobiledevice] build failure

**Host Environment**
- OS: Windows
- Compiler: Visual Studio 2019

**To Reproduce**
Steps to reproduce the behavior:
Install the package successfully: `./vcpkg install libimobiledevice:x64-windows`
When put `find_package(libimobiledevice REQUIRED)` in `CMakeLists.txt`, it fails: 
```
CMake Error at %vcpkg_root%/scripts/buildsystems/vcpkg.cmake:288 (_find_package):
1> [CMake]   By not providing "Findlibimobiledevice.cmake" in CMAKE_MODULE_PATH this
1> [CMake]   project has asked CMake to find a package configuration file provided by
1> [CMake]   "libimobiledevice", but CMake did not find one.
1> [CMake] 
1> [CMake]   Could not find a package configuration file provided by "libimobiledevice"
1> [CMake]   with any of the following names:
1> [CMake] 
1> [CMake]     libimobiledeviceConfig.cmake
1> [CMake]     libimobiledevice-config.cmake
1> [CMake] 
1> [CMake]   Add the installation prefix of "libimobiledevice" to CMAKE_PREFIX_PATH or
1> [CMake]   set "libimobiledevice_DIR" to a directory containing one of the above
1> [CMake]   files.  If "libimobiledevice" provides a separate development package or
1> [CMake]   SDK, be sure it has been installed.
Call Stack (most recent call first):
CMakeLists.txt:18 (find_package) 
-- Configuring incomplete, errors occurred! 
```

**Failure logs** 
```
1> [CMake] CMake Error at %vcpkg_root%/scripts/buildsystems/vcpkg.cmake:288 (_find_package):
1> [CMake]   By not providing "Findlibimobiledevice.cmake" in CMAKE_MODULE_PATH this
1> [CMake]   project has asked CMake to find a package configuration file provided by
1> [CMake]   "libimobiledevice", but CMake did not find one.
1> [CMake] 
1> [CMake]   Could not find a package configuration file provided by "libimobiledevice"
1> [CMake]   with any of the following names:
1> [CMake] 
1> [CMake]     libimobiledeviceConfig.cmake
1> [CMake]     libimobiledevice-config.cmake
1> [CMake] 
1> [CMake]   Add the installation prefix of "libimobiledevice" to CMAKE_PREFIX_PATH or
1> [CMake]   set "libimobiledevice_DIR" to a directory containing one of the above
1> [CMake]   files.  If "libimobiledevice" provides a separate development package or
1> [CMake]   SDK, be sure it has been installed.
```

**Additional context**
Normally after install, `%vcpkg_root%\installed\x64-windows\share\libimobiledevice` folder should contain some .cmake files, like libimobiledeviceConfig.cmake. But can't find any .cmake file in it.

Try remove and re-install the package, still have the same error.

--- a/ports/libimobiledevice/portfile.cmake
+++ b/ports/libimobiledevice/portfile.cmake
@@ -1,7 +1,7 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libimobiledevice-win32/libimobiledevice
-    REF 0d4a7e905baeadafa098e629a5241fac6fbf7d24 # v1.3.7
+    REF 0d4a7e905baeadafa098e629a5241fac6fbf7d24 # v1.3.17
     SHA512 db6369b2fa8e7b659948602ac8c4cd568bf37c0f73e58150f61f1af7000de95aa1a6a9ae546d6a37de4cb08a8059127fe5ed067351c4092c01a45350586f755a
     HEAD_REF msvc-master
 )
@@ -20,3 +20,4 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 # Handle copyright
 file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")

--- a/ports/libimobiledevice/usage
+++ b/ports/libimobiledevice/usage
@@ -1,3 +1,5 @@
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(LIBIMOBILEDEVICE REQUIRED IMPORTED_TARGET libimobiledevice-1.0)
-target_link_libraries(main PUBLIC PkgConfig::LIBIMOBILEDEVICE)
+libimobiledevice provides usage:
+
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(LIBIMOBILEDEVICE REQUIRED IMPORTED_TARGET libimobiledevice-1.0)
+    target_link_libraries(main PRIVATE PkgConfig::LIBIMOBILEDEVICE)

--- a/ports/libimobiledevice/usage
+++ b/ports/libimobiledevice/usage
@@ -1,0 +1,3 @@
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(LIBIMOBILEDEVICE REQUIRED IMPORTED_TARGET libimobiledevice-1.0)
+target_link_libraries(main PUBLIC PkgConfig::LIBIMOBILEDEVICE)

--- a/ports/libimobiledevice/vcpkg.json
+++ b/ports/libimobiledevice/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libimobiledevice",
   "version": "1.3.17",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A cross-platform protocol library to communicate with iOS devices",
   "homepage": "http://www.libimobiledevice.org",
   "license": "LGPL-2.1-only",
@@ -11,6 +11,7 @@
     "libplist",
     "libusbmuxd",
     "openssl",
+    "pkgconf",
     {
       "name": "vcpkg-cmake",
       "host": true

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3946,7 +3946,7 @@
     },
     "libimobiledevice": {
       "baseline": "1.3.17",
-      "port-version": 1
+      "port-version": 2
     },
     "libirecovery": {
       "baseline": "1.0.25",

--- a/versions/l-/libimobiledevice.json
+++ b/versions/l-/libimobiledevice.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "1e7fb241844889e7329fb9ee900092c25f86bf29",
+      "git-tree": "2945a66f7411d4b12ab5733be25abd5319a12718",
       "version": "1.3.17",
       "port-version": 2
     },

--- a/versions/l-/libimobiledevice.json
+++ b/versions/l-/libimobiledevice.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1e7fb241844889e7329fb9ee900092c25f86bf29",
+      "version": "1.3.17",
+      "port-version": 2
+    },
+    {
       "git-tree": "0f57daa63ba49e977a8e72eea7d3615cdda07648",
       "version": "1.3.17",
       "port-version": 1

